models.py

```diff
@@ -260,23 +269,28 @@ class PayDbTransaction(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     token = db.Column(db.String(255), unique=True, nullable=False)
-    timestamp = db.Column(db.Integer)
+    date = db.Column(db.DateTime())
     action = db.Column(db.String(255), nullable=False)
     sender_token = db.Column(db.String(255), db.ForeignKey('user.token'), nullable=False)
+    sender_id = db.Column(db.Integer)
     sender = db.relationship('User', foreign_keys=[sender_token], backref=db.backref('sent', lazy='dynamic'))
     recipient_token = db.Column(db.String(255), db.ForeignKey('user.token'), nullable=True)
+    recipient_id = db.Column(db.Integer)
```

Comment:
you should not need this, you can use `recipient_token`

Reply:
i need the query that will allow using both Filters "sender" and "recipient". Without my changes above.

Reply:
deleted teh recipient_id in the models. and have the filter that would search.